docs/hoon/twig/cen-call/sig-open.md

```diff
@@ -13,6 +13,10 @@ sort: 3
 :rap(p :make(q +6 r))
 ```
 
+```
+=<(p %=(q +6 r))
+```
+
```

Comment:
yeah, definitely.

My point is just that it wasn't clear to me, reading that expansion, that `q` needs to be a "reference" to an arm defined elsewhere, and it took a lot of trial and error to figure that out.